Dockerfile.rpi

```diff
@@ -2,9 +2,21 @@ FROM resin/rpi-node:0.10.40-slim
 
 COPY 01_nodoc /etc/dpkg/dpkg.cfg.d/
 
+ENV DOCKER_APT_VERSION 1.6.2~dfsg1-1~bpo8+1
+
 # Supervisor apt dependencies
-RUN apt-get -q update \
-	&& apt-get install -qqy socat supervisor --no-install-recommends \
+RUN echo deb "http://ftp.us.debian.org/debian jessie-backports main" >> /etc/apt/sources.list \
```

Comment:
@pcarranzav debian doesn't have armv6 builds, this won't work on the raspberry pi